README.md

```diff
@@ -27,26 +27,291 @@ Then make the necessary changes at the top of the file.
 ./bin/docker-clean.sh
 ```
 
-### Bringing it up
+## Getting Started
+
+The first time you set up, you will need to create the CCD network 
+
+```bash
+docker network create ccd-network
+```
+
+#### Using real idam (Default):
+
+Using idam for first time:
+
+```bash
+./bin/compose-up-idam.sh
+```
+
+Each subsequent time after you can just run this to restart the containers:
+
+```bash
+./ccd compose up -d
+```
+
+#### Using idam stub (Check setup instructions in section below before following this):
+
+Using the idam stub for first time:
+
+```bash
+./bin/compose-up-stub.sh
+```
+
+Each subsequent time after you can just run this to restart the containers:
+
+```bash
+./ccd compose up -d
+```
+
+Now import the CCD case definition locally. Please follow instructions in the sscs-ccd-definitions README. 
+
+Please read instructions below on how to switch between the real idam and idam stub
+
+#### Potential startup problems
+
+Network ccd-network declared as external, but could not be found error?
+Run `docker network create ccd-network` and try again.
+
+Authentication required error or error pulling images? try:
+
+```bash
+az acr login -n hmctspublic
+./ccd login
+```
+
+Pulling from sscs-logstash error? 
+Try opening the ccd-logstash project and running the `./bin/build-logstash-instances.sh` script.
+
+Error accessing CCD or XUI? 
+Ensure all environment variables are copied from .env.example to .env
```

Comment:
This should probably be a step 1, or a prerequisite to step 1, or perhaps highlighting (in bold) this more.
For this change I overwrote the file from the .env.example and had to fill out the last two docmosis keys

Reply:
It already is at the top of the Readme under the Quick start section. I can try and make clearer...

Reply:
Updated